From c4786579bbf332695f113c5bf64b31bf7f37e3ee Mon Sep 17 00:00:00 2001
From: sreesurendran <sree.surendran55@gmail.com>
Date: Tue, 7 Oct 2014 01:06:40 -0500
Subject: [PATCH] Fifth Commit

---
 HelloWorld.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/HelloWorld.java b/HelloWorld.java
index b316ea1..e31d929 100644
--- a/HelloWorld.java
+++ b/HelloWorld.java
@@ -1,7 +1,7 @@
 
 public class HelloWorld {
 	public static void main( String [] args){
-		System.out.println("Hello World!");
+		System.out.println("Bonjour, monde");
 	}
 
 }
-- 
1.9.3 (Apple Git-50)

